src/signals/incident/components/form/MapSelectors/Streetlight/StreetlightLayer/StreetlightLayer.test.tsx

```diff
@@ -22,7 +22,7 @@ const assetSelectProviderValue: AssetSelectValue = {
     id: '031346',
     type: '4',
     description: 'Overig lichtpunt',
-    isReported: true,
+    status: 'reported',
```

Comment:
Can we have the possible values for the `status` prop stored in a type, so that it can be validated?

Reply:
done that